ServiceLayer/src/main/java/org/example/ProductCreationService.java

```diff
@@ -0,0 +1,17 @@
+package org.example;
```

Comment:
Why are the services present in org.example package. They should be present in servicelayer package ?